RubeesEat/Model/DB/DbBillRepository.cs

```diff
@@ -110,7 +110,11 @@ public PaginatedView<BalanceChange> GetRecentBalanceChanges(Person currentUser,
                                                           SUM(el.Amount) AS TotalAmount, 
                                                           b.Date AS Date,
                                                           b.Description AS Description,
-                                                          b.BillID AS BillID
+                                                          b.BillID AS BillID,
+                                                          (SELECT COUNT(*) 
+                                                           FROM EntryLines el2 
+                                                           WHERE el2.PersonID = @PersonID
+                                                          ) AS TotalEntryLines
```

Comment:
I am not sure that this has to be so complex. Can you test if `COUNT(*) OVER ()` works as well?